fpdf/util.py

```diff
@@ -41,6 +42,54 @@ def b(s):
     raise ValueError(f"Invalid input: {s}")
 
 
+def get_scale_factor(unit: Union[str, float, int]) -> float:
+    """
+    Get how many pts are in a unit. (k)
+
+    Args:
+        unit (str, float, int): Any of "pt", "mm", "cm", "in", or a number
```

Comment:
Same remark: could you please explain in docstring how a numeral value is interpreted / used by this function?
Judging from the code, a numeral value would directly represent a number of points, right?

Reply:
Done